custom/icds_reports/sqldata.py

```diff
@@ -2232,7 +2232,8 @@ def new_table_config(self):
                                 'header': 'Percent of children born in month with low birth weight',
                                 'slug': 'low_birth_weight',
                                 'average': [],
-                                'format': 'percent'
+                                'format': 'percent',
+                                'reverseColors': True,
```

Comment:
what does reverseColors do?

Reply:
@emord 
This change arrow colors, if ```reverseColors``` is ```True``` then up arrow will be red and down arrow will be green